### PR TITLE
Fixes for ERC-1155 with evmconnect

### PR DESCRIPTION
**Pull request by nguyer**
This PR fixes ERC-1155 to make it compatible with both Ethereum blockchain connectors: ethconnect and evmconnect.

> **NOTE**: Due to how the ERC-1155 connector currently expects a specific contract to be deployed to the chain before it is set up, it is currently not compatible with `remote-rpc` blockchain nodes.

### CODE REVIEW

**Comment by awrichar** [comment]
I'm happy to hardcode the base URI in the ERC-1155 sample contract if that's the easiest step for the short term.

File: internal/stacks/stack_manager.go [hunk 1]
```diff
@@ -505,15 +509,23 @@ func (s *StackManager) createMember(id string, index int, options *types.InitOpt
...
-		ExposedIPFSGWPort:          serviceBase + 7,
 		External:                   external,
 		OrgName:                    options.OrgNames[index],
 		NodeName:                   options.NodeNames[index],
 	}
-	nextPort := serviceBase + 8
+
+	nextPort := serviceBase + 4
+	if options.MultipartyEnabled {
+		member.ExposedDatabasePort = serviceBase + nextPort
```
Comment (awrichar) [inline internal/stacks/stack_manager.go:519]: The database will exist for multiparty and non-multiparty.

**Review by awrichar — APPROVED** [review]
Noted one thing I think needs to change, but other than that looks good. I was able to run E2E tests locally with this and the current HEAD from erc1155 and evmconnect.

**Comment by awrichar** [comment]
Also noting that it looks like this has been updated to support passing the args to the ERC1155 contract constructor.

File: internal/stacks/stack_manager.go (hunk 1 above)
Comment (nguyer) [reply internal/stacks/stack_manager.go:519]: Thanks good catch

**Pull request merged**